src/jsonrpc/miner_jsonrpc_ledger.erl

```diff
@@ -166,12 +168,21 @@ get_ledger() ->
             blockchain:ledger(Chain)
     end.
 
-format_ledger_balance(Addr, Entry) ->
-    #{
-        <<"address">> => ?BIN_TO_B58(Addr),
-        <<"nonce">> => blockchain_ledger_entry_v1:nonce(Entry),
-        <<"balance">> => blockchain_ledger_entry_v1:balance(Entry)
-    }.
+format_ledger_balance(Addr, Entry, Ledger) ->
+    case blockchain_ledger_v1:config(token_version, Ledger) of
```

Comment:
You could call `blockchain_ledger_v1:versioned_entry_mod_and_entries_cf/1` and not have to do the token_version check here by hand. So this function can be reduced to something like this:

```erlang
{EntryMod, _} = blockchain_ledger_v1:versioned_entry_mod_and_entries_cf(Ledger),
#{<<"nonce">> = EntryMod:nonce(Entry), ...} 
```